plugins/builds/get.js

```diff
@@ -23,27 +23,40 @@ module.exports = () => ({
             }
         },
         handler: (request, reply) => {
-            const factory = request.server.app.buildFactory;
+            const buildFactory = request.server.app.buildFactory;
+            const stepFactory = request.server.app.stepFactory;
 
-            return factory.get(request.params.id)
-                .then((model) => {
-                    if (!model) {
+            return Promise.all([buildFactory.get(request.params.id),
+                stepFactory.list({ params: { buildId: request.params.id }, sortBy: 'id', sort: 'ascending' })])
+                .then(([buildModel, stepsModel]) => {
+                    if (!buildModel) {
                         throw boom.notFound('Build does not exist');
                     }
 
-                    if (Array.isArray(model.environment)) {
-                        return reply(model.toJson());
+
+                    // This if statement should be removed after enough time has passed since build.steps removed.
+                    // Make orders of steps in completed builds sure,
+                    // because steps in old builds in DB have the order not sorted.
+                    if (buildModel.endTime) {
+                        stepsModel.sort((a, b) => new Date(a.startTime).getTime() - new Date(b.startTime).getTime());
+                    }
```

Comment:
Do we need to add the test case about these codes?

Reply:
Thank you.
added test

Reply:
Would you mind adding some test code for the case where `stepsModel` are in the order of `id` when `buildModel.endTime` is undefined?